src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx

```diff
@@ -1362,6 +1362,38 @@ For agent versions 4.10.0 or higher the following environment variables are avai
 
     The unqualified log file name or the string `STDOUT` which will log to standard out.
   </Collapser>
+  
+  <Collapser
+    id="NEW_RELIC_METADATA_SERVICE_VERSION"
+    title="NEW_RELIC_METADATA_SERVICE_VERSION"
+  >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            String
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+             (none)
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    
+  Setting NEW_RELIC_METADATA_SERVICE_VERSION will create the tag, tag.service.version on error data. 
```

Comment:
We need to either explain here what the service.version is and what value setting it gives the user or link to a place that does.

Reply:
It should say event data not just error data.